plexupdate-core

```diff
@@ -226,17 +226,23 @@ isNewerVersion() {
 }
 
 parseVersion() {
-	if [ "${REDHAT}" = "yes" ]; then
+	if [ "${DISTRO}" = "redhat" ]; then
 		cut -f2- -d- <<< "$1" | cut -f1-4 -d.
+	elif [ "${DISTRO}" = "synology" -o "${DISTRO}" = "synology-dsm7" ]; then
+		cut -f2-3 -d- <<< "$1"
```

Comment:
The output of `synopkg` appears to have changed, so I think the correct cut now is `cut -f1 -d-`
```
$ synopkg version "PlexMediaServer"
1.24.0.4930-7000
$ synopkg version "PlexMediaServer" | cut -f1 -d-
1.24.0.4390
```